<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTFCodecTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageEncode">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>560</height>
        </rect>
       </property>
       <attribute name="label">
        <string>编码</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageEncrypt">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>560</height>
        </rect>
       </property>
       <attribute name="label">
        <string>古典加密</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageHex">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>560</height>
        </rect>
       </property>
       <attribute name="label">
        <string>进制转换</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageHash">
       <attribute name="label">
        <string>Hash</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_src">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0">
          <item>
           <widget class="QLineEdit" name="srcLineEdit1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>参数1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="srcLineEdit2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>参数2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="srcReplaceButton">
            <property name="text">
             <string>替换</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="srcClearButton">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="srcCopyButton">
            <property name="text">
             <string>复制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="srcPasteButton">
            <property name="text">
             <string>粘贴</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="srcOpenButton">
            <property name="text">
             <string>打开文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="srcTextEdit">
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_dst">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="dstLineEdit1">
            <property name="placeholderText">
             <string>参数3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dstLineEdit2">
            <property name="placeholderText">
             <string>参数4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="swapButton">
            <property name="text">
             <string>上下交换</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dstReplaceButton">
            <property name="text">
             <string>替换</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dstClearButton">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dstCopyButton">
            <property name="text">
             <string>复制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dstPasteButton">
            <property name="text">
             <string>粘贴</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dstSaveButton">
            <property name="text">
             <string>保存文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="dstTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageDecode">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>612</height>
        </rect>
       </property>
       <attribute name="label">
        <string>解码</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageDecrypt">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>612</height>
        </rect>
       </property>
       <attribute name="label">
        <string>古典解密</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEncode">
    <property name="title">
     <string>编码</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDecode">
    <property name="title">
     <string>解码</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEncrypt">
    <property name="title">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDecrypt">
    <property name="title">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHex">
    <property name="title">
     <string>进制转换</string>
    </property>
   </widget>
   <addaction name="menuEncode"/>
   <addaction name="menuDecode"/>
   <addaction name="menuEncrypt"/>
   <addaction name="menuDecrypt"/>
   <addaction name="menuHex"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dstClearButton</sender>
   <signal>clicked()</signal>
   <receiver>dstTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srcClearButton</sender>
   <signal>clicked()</signal>
   <receiver>srcTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dstPasteButton</sender>
   <signal>clicked()</signal>
   <receiver>dstTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srcPasteButton</sender>
   <signal>clicked()</signal>
   <receiver>srcTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
